### PR TITLE
[core][idea] #196 Rebranding BetterDocs as KodeBeagle.

### PR DESCRIPTION
**Pull request by ScrapCodes**

### CODE REVIEW

**Comment by ScrapCodes** [comment]
Review @prudhvije @amit-shar

**Comment by ScrapCodes** [comment]
@PiyushMishra PTAL

File: plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -57,8 +57,8 @@
     </actions>
 
     <extensions defaultExtensionNs="com.intellij">
-        <toolWindow id="Better Docs" icon="AllIcons.Toolwindows.Documentation" anchor="right" factoryClass="com.imaginea.betterdocs.ui.MainWindow"/>
-        <applicationConfigurable instance="com.imaginea.betterdocs.ui.SettingsPanel"/>
+        <toolWindow id="Better Docs" icon="AllIcons.Toolwindows.Documentation" anchor="right" factoryClass="com.imaginea.kodebeagle.ui.MainWindow"/>
```
Comment (prudhvije) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: toolwindow id?

Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: Can I make it KodeBeagle ? Or it should be "Kode Beagle"

Comment (prudhvije) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: Other plugins are following like this:
Maven is named as "Maven Projects" 
Ant is named as "Ant Build"
Shall we also follow the same?

Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: Ok Agreed !

File: plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -1,17 +1,17 @@
 <idea-plugin version="3">
-  <id>betterdocsidea</id>
-  <name>Better Docs Plugin for Intellij Idea</name>
+  <id>kodebeagleidea</id>
+  <name>Kode Beagle Plugin for Intellij Idea</name>
```
Comment (prudhvije) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: Can we change this to Kode Beagle? cause it's looking very lengthy in plugin manger window of idea.

image

Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/resources/META-INF/plugin.xml]: Good point !

**Pull request closed without merging**